### ISSUE TITLE
Default value for input argument not bound to schema

### ISSUE THREAD

**Issue by thebenwaters** [mirumee/ariadne#321]
I have a schema like the following

```
enum FooEnum {
   FOO
   BAR
}
input FooBarInput {
    foo: FooEnum! = FOO
}

extend type mutation {
foo(input: FooBarInput): Boolean!
}
```
But when calling the mutation FOO is not passed as an argument and the resolver fails

**Comment by rafalp** [comment]
Duplicate of #321

**Comment by rafalp** [comment]
Ok, not exact duplicate, but same source and same fix, so I've updated #293 instead to track both of those. Thanks!
